microsoft-365/managed-desktop/service-description/updates.md

```diff
@@ -32,12 +32,18 @@ Microsoft Managed Desktop uses four Azure AD groups to manage updates:
 - **Fast**: Prioritizes speed over stability. Useful for detecting quality issues before they are offered to the Broad group. This group serves as a next layer of validation but is generally more stable than the Test and First groups. 
 - **Broad**: Last group to have feature and quality updates available. This group contains the majority of users in the tenant, and therefore favors stability over speed in deployment. Testing of apps should be done here as the environment is most stable. 
 
+### Moving Devices between Update Groups
+You may want some devices to receive updates last and others that you want to go first. To move these devices into the appropriate Update Group [submit an administrator support request](https://docs.microsoft.com/en-us/microsoft-365/managed-desktop/working-with-managed-desktop/admin-support?view=o365-worldwide) and we will move the devices for you. 
+
 > [!NOTE]
-> If you need to move a user to a different update group, submit a support request. See [Support for Microsoft Managed Desktop](support.md) for more information on submitting support requests. If you move a user yourself, the move will be reverted.
+> If you need to move a user to a different update group, submit a support request. Do not move devices between update groups yourself, there are serious consequences if a device is moved incorrectly: the device may update unexpectedly and policies may conflict, changing the device configuration.
 
 For more information roles and responsibilities with these deployment groups, see [Microsoft Managed Desktop Roles and responsibilities](../intro/roles-and-responsibilities.md)
 
-How update deployment works:
+### Using Microsoft Managed Desktop Update Groups 
+There are parts of the service, like app deployment, which you manage where it may be necessary to target all managed devices. In these instances, it makes sense to use the Update Groups to reach those users with the understanding that you cannot add, remove, or change the membership of those groups. 
```

Comment:
```suggestion
There are parts of the service, like app deployment, which you manage where it may be necessary to target all managed devices. In these instances, it makes sense to use the update groups to reach those users with the understanding that you cannot add, remove, or change the membership of those groups. 
```